dicoogle/src/main/java/pt/ua/dicoogle/server/users/UserExportPresets.java

```diff
@@ -43,19 +44,11 @@
      * @throws IOException if an I/O error occurs
      */
     public static void savePreset(String username, String presetName, String[] fields) throws IOException {
-        File presetsDir = new File(Platform.homePath() + "users/" + username + "/presets");
-
-        // create presets dir, if it doesn't exist
-        Files.createDirectories(presetsDir.toPath());
+        Path presetsDir = Paths.get(Platform.homePath(), "userdata", username, "presets");
 
-        File presetFile = new File(presetsDir + "/" + presetName + ".txt");
+        Files.createDirectories(presetsDir);
 
-        try (Writer writer = new PrintWriter(presetFile)) {
-            for (String field : fields) {
-                writer.write(field);
-                writer.write('\n');
-            }
-        }
+        Files.write(presetsDir.resolve(presetName), Arrays.asList(fields));
```

Comment:
Oh, by the way, these files could be .txt again.

```suggestion
        Files.write(presetsDir.resolve(presetName + ".txt"), Arrays.asList(fields));
```